cukinia

```diff
@@ -402,46 +402,52 @@ _cukinia_mount()
 # arg2..: string
 _colorize()
 {
-	local color="$1"; shift
-	local text="$*"
-	local nc='\033[0m'
-	local c
-
-	# Only colorize a few terminal types
-	case "$TERM" in
-	linux*|xterm*|vt102)
-		;;
-	*)
-		echo "$@"
-		return
-		;;
-	esac
+	if [ -z ${__log_format} ] && [ $__log_file -e "&2" ];
```

Comment:
* invert your test: that way you can return in the simple case, and shift the rest of the funcion 1 tab left
* put the string variables you test between "quotes" eg. `"$__log_format"
* you don't need `${ }` variable protection with $__log_format
* "-e &2" is a string not a file name, can't you just leave it empty if we want to log to stdout?